playwright/pages/dialogs/addNavigationMenuItemDialog.ts

```diff
@@ -15,8 +15,11 @@ export class AddNavigationMenuItemDialog extends BasePage {
   }
 
   async selectLinkTypeOption(linkType: string) {
-    await this.menuLinkType.click();
-    await this.waitForDOMToFullyLoad();
+    await expect(async () => {
+      await this.menuLinkType.click();
+      await this.waitForDOMToFullyLoad();
```

Comment:
suggestion: Do you need it, it only works on first-page loading. Someone who does not know how it works puts it everywhere, and no one knows why.

Reply:
@poulch  I removed all `waitForDOMToFullyLoad` from this file and used `expect.toPass` also for selecting the value